inventory/aws/ec2.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
We need it on bleeding edge distributions on which `python` is a symlink to `python3`.
On my Arch Linux, without that patch and with `#!/usr/bin/env python`, I get the following error:
```
    print data_to_print
                      ^
SyntaxError: Missing parentheses in call to 'print'
```

The full story has been discussed on Ansible’s github there:
https://github.com/ansible/ansible/issues/2974

In short, for the distributions where `python` is `python3`,
* the install script will take care of patching the shebang,
* the version of the interpreter used on minions can be overridden thanks to `ansible_python_interpreter`
* But as this file is shipped directly in this repo, it gets no chance to be "fixed".